cms/djangoapps/contentstore/helpers.py

```diff
@@ -618,8 +661,8 @@ def _import_file_into_course(
         contentstore().save(content)
         return True, {clipboard_file_path: f"static/{import_path}"}
     elif current_file.content_digest == file_data_obj.md5_hash:
-        # The file already exists and matches exactly, so no action is needed
-        return None, {}
+        # The file already exists and matches exactly, so no action is needed except substitutions
+        return None, {clipboard_file_path: f"static/{import_path}"}
```

Comment:
library sync was failing when updating downstream blocks with the following button
![image](https://github.com/user-attachments/assets/1841dad2-daa1-4723-bf1b-a482204c98c1)